docs/tools/drain.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+#
+# This file is part of MinIO DirectPV
+# Copyright (c) 2023 MinIO, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#
+# This script drains the DirectPV resources from a selected node
+# 
+# **CAUTION**
+#
+# This operation is irreversible and may incur data loss if not used cautiously.
```

Comment:
1. Add this caution message to `usage` as well.
2. Add `set -e -C -o pipefail` and fix the script accordingly.